ingestion/strip_data.py

```diff
@@ -4,74 +4,66 @@
 import logging
 import sys
 
-def austin():
+def austin() -> pd.DataFrame:
     """
     Extracts needed columns from the Austin data and saves it as a csv file
     :return austin_stripped: the stripped data as a pandas dataframe
     """
-    austin_raw = pd.read_csv("./raw_data/austin.csv")
+    AUSTIN_RAW_PATH = "./raw_data/austin.csv"
+    AUSTIN_STRIPED_PATH = "./stripped_data/austin.csv"
+    austin_raw = pd.read_csv(AUSTIN_RAW_PATH)
     austin_stripped = austin_raw[austin_raw.contractor_trade == "Electrical Contractor"]
     austin_stripped = austin_stripped[["issue_date", "location", "contractor_company_name"]]
-    latitude = []
-    longitude = []
-    for i in range(len(austin_stripped) - 1):
-        # {'latitude': '30.29406429', 'longitude': '-97.69323996', 'human_address': '{""address"": """", ""city"": """", ""state"": """", ""zip"": """"}'}
-        location_object = austin_stripped["location"][i]
-        if type(location_object) != str:
-            latitude.append("nan")
-            longitude.append("nan")
-            continue
-        latitude_temp, longitude_temp = location_object.split(",")[:2]
-        latitude.append(latitude_temp.split("'")[3])
-        longitude.append(longitude_temp.split("'")[3])
-    austin_stripped["latitude"] = pd.Series(latitude)
-    austin_stripped["longitude"] = pd.Series(longitude)
+    # {'latitude': '30.29406429', 'longitude': '-97.69323996', 'human_address': '{""address"": """", ""city"": """", ""state"": """", ""zip"": """"}'}
+    austin_stripped["latitude"] = austin_stripped["location"].apply(lambda x: x.split(",")[0].split("'")[3] if type(x) == str else "nan")
```

Comment:
Should be `else np.nan` in all these places and not "nan"

Where np is `import numpy as np`